appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  # The CONDA_PY is just to ensure we will test with vc 9, vc 10, and vc 14.
+  matrix:
+    - TARGET_ARCH: x64
+      CONDA_PY: 27
```

Comment:
@kwilcox I guess that `pocean-core` is not Python 2.7 compatible. I removing this...